nntrainer/tensor/meson.build

```diff
@@ -34,6 +35,7 @@ arch = target_machine.cpu_family()
 
 if get_option('platform') == 'android' or arch == 'aarch64'
   tensor_sources += 'blas_neon.cpp'
+  tensor_sources += 'half_tensor.cpp'
```

Comment:
I think this is added when enable_fp16 is true.

Reply:
Applied!